scene/gui/link_button.cpp

```diff
@@ -273,27 +291,20 @@ void LinkButton::_get_property_list(List<PropertyInfo> *p_list) const {
 void LinkButton::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_text", "text"), &LinkButton::set_text);
 	ClassDB::bind_method(D_METHOD("get_text"), &LinkButton::get_text);
-	ClassDB::bind_method(D_METHOD("set_text_direction", "direction"), &LinkButton::set_text_direction);
-	ClassDB::bind_method(D_METHOD("get_text_direction"), &LinkButton::get_text_direction);
-	ClassDB::bind_method(D_METHOD("set_opentype_feature", "tag", "value"), &LinkButton::set_opentype_feature);
```

Comment:
Rebase error; this should be added back (this is part of the Complex Text Layouts work).